src/commands/runtime/action/create.js

```diff
@@ -144,6 +144,18 @@ class ActionCreate extends RuntimeBaseCommand {
           case 'no':
             annotationParams.push({ key: 'web-export', value: false })
         }
+
+        if (flags['web-secure']) {
```

Comment:
We could check for `--web` not being `false` here and throw an error if it is. Also add a test for the same.

Reply:
OK - this is reasonable. The flag is dependent on `--web` but not necesarily `--web true|string`

Reply:
Actually @Himavanth this predicate is guarded by `if (flags.web)` on [line 128](https://github.com/adobe/aio-cli-plugin-runtime/pull/220/files#diff-536d7d7870c9282c29c1bae32f929f3248b84dc11d5c664dfe1753875aef9abcR128). So a check for web === false and web-secure !== false would have to come sooner. Is the existing predicate sufficient or did you intend for the CLI to fail?

Reply:
You are right @rabbah . It would have to come sooner (or in an else statement). I had intended for an error as `wsk` seems to throw an error and `aio cli` in this case just exits silently giving a wrong impression. WDYT ?

Reply:
Addressed in the last commit.

Reply:
Thx for the changes @rabbah .